games-board/xboard/xboard-4.9.0.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools eutils fdo-mime gnome2-utils
```

Comment:
please switch from `fdo-mime` to `xdg-utils` and drop unused `eutils`, also keep one empty line after `EAPI`.